ingest/src/main/java/com/microsoft/azure/kusto/ingest/source/BlobSourceInfo.java

```diff
@@ -29,20 +31,24 @@ public void setRawSizeInBytes(long rawSizeInBytes) {
 
     public BlobSourceInfo(String blobPath) {
         this.blobPath = blobPath;
+        this.blobPathWithoutSecrets = blobPath.split(";",2)[0].split("[?]",2)[0];
```

Comment:
Isn't it the same to split by the "[?]" without the first part?

```suggestion
        this.blobPathWithoutSecrets = blobPath.split("[?]",2)[0];
```

Reply:
i guess u r right 
i took it from the .Net implementation - kusto replaces the '?' with ';' ... but its not relevant here